src/RequestFactory.hack

```diff
@@ -22,7 +22,7 @@ final class RequestFactory implements RequestFactoryInterface {
     return new Request(
       $method,
       $uri,
-      IO\server_input(),
+      IO\request_input(),
```

Comment:
`server_input()` was removed but `request_input()` is stdin for script which is probably what we want here...?

I wasn't sure since it looks like this isn't actually used anywhere.

Reply:
yeah, this should be using the request_*